swagger.yaml

```diff
@@ -148,19 +401,164 @@ paths:
           description: ID of the run
           required: true
       responses:
-        200:
+        '200':
           description: Sends the entries for a single run
           schema:
-            type: object
             $ref: '#/definitions/EntryList'
-        404: 
+        '401':
+          $ref: '#/responses/UnauthorizedError'
+        '404':
           description: indicates entries are not yet posted
-        401:
-          $ref: "#/responses/UnauthorizedError"              
-
-  /events/{event_id}/runs/{run_id}/racers/{racer_id}:
+        '429':
+          $ref: '#/responses/RetryLaterError'
+    post:
+      summary: Create a run with racers.
+      parameters:
+        - in: path
+          name: event_id
+          type: integer
+          description: ID of the event
+          required: true
+        - name: run_id
+          in: path
+          type: integer
+          description: ID of the run
+          required: true
+        - in: body
+          name: body
+          description: The attributes and racers you want to add to the run.
+          schema:
+            type: object
+            properties:
+              racers:
+                $ref: '#/definitions/Racers'
+      responses:
+        '200':
+          description: Updates the group with the provided racers.
+          schema:
+            $ref: '#/definitions/EntryList'
+        '401':
+          $ref: '#/responses/UnauthorizedError'
+        '404':
+          description: indicates group does not exist
+        '429':
+          $ref: '#/responses/RetryLaterError'
+  '/events/{event_id}/runs/{run_id}/flags':
+    get:
+      summary: Get all flags for a run.
+      parameters:
+        - name: event_id
+          in: path
+          type: integer
+          description: ID of the event
+          required: true
+        - name: run_id
+          in: path
+          type: integer
+          description: ID of the run
+          required: true
+      responses:
+        '200':
+          description: get the flags of a single run
+          schema:
+            $ref: '#/definitions/Flag'
+        '401':
+          $ref: '#/responses/UnauthorizedError'
+        '429':
+          $ref: '#/responses/RetryLaterError'
+    post:
+      summary: Add a flag to a run.
+      parameters:
+        - in: path
+          name: event_id
+          type: integer
+          description: ID of the event
+          required: true
+        - name: run_id
+          in: path
+          type: integer
+          description: ID of the run
+          required: true
+        - in: body
+          name: body
+          description: Flag type and timestamp.
+          schema:
+            type: object
+            properties:
+              name:
```

Comment:
@chrisredrich I think this should be an enum of possible flag types, no?

Reply:
@tejaswinichile Brian is right here. Could you change this to be an enum with the following flag options? 

"WarmUp Flag",
"Green Flag",
"Yellow Flag",
"Red Flag",
"Stop Flag",
"Finish Flag"

Reply:
@chrisredrich from a developer joy perspective, should we fix these before we release them into the wild? I think it would be nicer if they were: 

- warmup
- green
- yellow
- red
- finish
- stop

single word, all lower case, and we can translate internally to what we need to translate?